src/quemb/molbe/be_parallel.py

```diff
@@ -92,19 +92,25 @@ def run_solver(
         Default is 'MP2'.
     eri_file :
         Filename for the electron repulsion integrals. Default is 'eri_file.h5'.
+    veff :
+        Veff matrix to be passed to energy, if non-cumulant energy.
+    veff0_per :
+        Veff0 matrix, passed to energy for periodic calculations
```

Comment:
As discussed with @lweisburn in-person, these two arguments are necessary for now because of the mismatch in the `pFrag.veff0` attribute between the molecular and periodic code. This will (and should be) fixed in the future as a more fundamental fix, because we rely on this attribute for other projects. I will create an issue for this.